shared/chat-sdk/reference/release-notes/unity.mdx

```diff
@@ -1,5 +1,68 @@
 <PlatformWrapper platform="unity">
 
+## v1.0.9
+
+v1.0.9 was released on February 1, 2023.
+
+#### New features
+
+1. Adds the following methods to the `SDKClient` class:     
+  - `GetLoggedInDevicesFromServer`: Gets the list of online devices on which you have logged in with a specified account.
```

Comment:
The rest of release notes are using the word "user", not "you". Can this please be made consistent?